src/background/responders.ts

```diff
@@ -89,8 +89,12 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
       },
     }
   },
-  START_FETCH_HANDLE(state): Partial<AppState> {
-    const tab = ensureActiveTab(state)
+  FETCH_HANDLE_START(state, action): Partial<AppState> {
+    const { tabId } = action.payload
```

Comment:
I assumed that `FETCH_HANDLE_START` is called when the tab is opened. That’s why I used `ensureActiveTab` and didn’t check if the tab exists. If we say that we can’t be sure that the tab exists anymore, then all the other actions can’t use `ensureActiveTab` either.